homeassistant/components/switchbot/sensor.py

```diff
@@ -71,14 +71,16 @@ async def async_setup_entry(
 ) -> None:
     """Set up Switchbot sensor based on a config entry."""
     coordinator: SwitchbotDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
-    async_add_entities(
+    entities = [
         SwitchBotSensor(
             coordinator,
             sensor,
         )
         for sensor in coordinator.data["data"]
         if sensor in SENSOR_TYPES
```

Comment:
Is this going to include `rssi` too ?

Reply:
Nope since its removed in the upstream library (which is why I bumped the version since its technically a breaking change in the upstream lib)